### PR TITLE
Add Basket to replace BSD

### PR DESCRIPTION
**Pull request by alicoding**
This patch basically added Basket support event to replace BSD

### CODE REVIEW

**Comment by alicoding** [comment]
@cadecairos

File: worker/sign_up_for_webmaker_mailing_list.js [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+        data: {
+          email: event.data.email,
+          lang: event.data.locale
+        }
+      }, cb);
+    } else {
+      process.nextTick(cb);
+    }
+  }
+}
```
Comment (cadecairos) [inline worker/sign_up_for_webmaker_mailing_list.js]: semicolon plz

File: worker/sign_up_for_webmaker_mailing_list.js [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+        event_type: NOTIFIER_EVENT_TYPE,
+        data: {
+          email: event.data.email,
+          lang: event.data.locale
+        }
+      }, cb);
+    } else {
+      process.nextTick(cb);
+    }
+  }
```
Comment (cadecairos) [inline worker/sign_up_for_webmaker_mailing_list.js]: semicolon

**Comment by cadecairos** [comment]
Travis failures are present on master, and not related to this.

**Pull request merged**